target/linux/ath79/tiny/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -9,46 +9,9 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath9k-eeprom-pci-0000:00:00.0.bin")
 	case $board in
-	buffalo,whr-g301n|\
-	engenius,eap350-v1|\
-	engenius,ecb350-v1|\
-	engenius,enh202-v1|\
-	tplink,tl-wa701nd-v1|\
-	tplink,tl-wa730re-v1|\
-	tplink,tl-wa801nd-v1|\
-	tplink,tl-wa830re-v1|\
-	tplink,tl-wa901nd-v1|\
-	tplink,tl-wr841-v5|\
-	tplink,tl-wr941-v4)
-		caldata_extract "art" 0x1000 0xeb8
-		;;
 	dlink,dir-615-e4)
-		caldata_extract "art" 0x1000 0x1000
 		ath9k_patch_mac_crc $(mtd_get_mac_ascii "nvram" "lan_mac") 0x10c
```

Comment:
This also needs to be converted to **10_fix_wifi_mac**.

Reply:
there's no 10_fix_wifi_mac for tiny.

Reply:
You can create it.

Reply:
Done.